### ISSUE TITLE
Inconsistency between `reporx-find-data` and `reprox-submit-jobs` in argument naming

### ISSUE THREAD

**Issue by FaroutYLq** [XENONnT/reprox#110]
```
reprox-find-data --context xenonnt_online --config-kwargs '{"minimum_run_number":51379,"maximum_run_number":52358,"_rucio_path":"/project/lgrandi/rucio","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cut_daq_vet
```
This will lead to `TypeError: get_context() got an unexpected keyword argument '_rucio_path'`. To make it work, you need to do this instead:
```
reprox-find-data --context xenonnt_online --context-kwargs '{"minimum_run_number":51379,"maximum_run_number":52358,"_rucio_path":"/project/lgrandi/rucio","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cut_daq_vet
```
However, this will fail:
```
reprox-start-jobs --context xenonnt_online --context-kwargs '{"minimum_run_number":51379,"maximum_run_number":52358,"_rucio_path":"/project/lgrandi/rucio","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cut_daq_veto
```
To make it work, you need to do 
```
reprox-start-jobs --context xenonnt_online --config-kwargs '{"minimum_run_number":51379,"maximum_run_number":52358,"_rucio_path":"/project/lgrandi/rucio","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cut_daq_veto
```

**Comment by FaroutYLq** [comment]
Solved in 8f1a65d7be204be26a8ffd6d3c7b03d47d8020d5
Now this will work:
```
~/software/reprox/bin/reprox-start-jobs --context xenonnt_online --context-kwargs '{"minimum_run_number":51379,"maximum_run_number":52358,"_rucio_path":"/project/lgrandi/rucio","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cut_daq_veto
~/software/reprox/bin/reprox-find-data--context xenonnt_online --context-kwargs '{"minimum_run_number":51379,"maximum_run_number":52358,"_rucio_path":"/project/lgrandi/rucio","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cut_daq_veto
```

**Comment by FaroutYLq** [comment]
Solved in #114 
Now the following works:
```
~/software/reprox/bin/reprox-find-data --context xenonnt_onlin
e --config-kwargs '{"_rucio_local_path":"/project/lgrandi/rucio","include_rucio_local":"true","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cuts_basic
~/software/reprox/bin/reprox-start-jobs --context xenonnt_onlin
e --config-kwargs '{"_rucio_local_path":"/project/lgrandi/rucio","include_rucio_local":"true","output_folder":"/project2/lgrandi/xenonnt/processed"}' --targets cuts_basic
```
